contributing/topics/reference-naming.md

```diff
@@ -8,10 +8,29 @@ Here are some general guidelines you can turn to when naming properties:
 
 * The name should describe what the property is for succinctly, but as with many things a balance should be struck between too short or too long.
 
-* Abbreviations should not be used and the full words should be used instead e.g. `resource_group_name` instead of `rg_name` or `virtual_machine` instead of `vm`. 
+* Choose the officially marketed name for new properties over the ones used in the API if they differ.
 
-* For blocks avoid redundant words in the name that don't add informational value e.g.`firewall_properties` can be shortened to `firewall`, the same can apply to individual properties e.g. `email_address` to `email`.
+* Abbreviations should not be used and the full words should be used instead e.g. 
+>`resource_group_name` instead of `rg_name` or `virtual_machine` instead of `vm`.
 
-* As a general rule, booleans should be appended with `_enabled`, e.g. `public_network_access_enabled`.
+* For blocks avoid redundant words in the name that don't add informational value e.g.
+>`firewall_properties` can be shortened to `firewall`, the same can apply to individual properties e.g. `email_address` to `email`.
 
-* Choose the officially marketed name for new properties over the ones used in the API if they differ.
\ No newline at end of file
+* Properties for certificates or artifacts that must be in a certain format should be appended with the format e.g.
+> A certificate that must be base64 encoded should be named `certificate_in_base64`
+
+* Similarly, properties that pertaining to sizes or times should be appended with the appropriate unit of measure e.g.
```

Comment:
Times themselves might be worth a separate note/callout? These should assume RFC3339 unless called out as `utc`, iirc?